src/app/components/applications/SSV/MyAccount/components/Withdraw/Withdraw.tsx

```diff
@@ -29,7 +29,7 @@ const Withdraw = () => {
     if (processStore.isValidatorFlow) {
       interval = setInterval(async () => {
         const balance = await getClusterBalance(processItem.operators, walletStore.accountAddress, ssvStore.liquidationCollateralPeriod, ssvStore.minimumLiquidationCollateral);
-        setProcessItemBalance(balance);
+        setProcessItemBalance(fromWei(balance));
```

Comment:
you can move it conditionally to getClusterBalance.
Add to its' sig inWei: boolean

Reply:
done